en/orm/saving-data.rst

```diff
@@ -147,11 +147,17 @@ one or many entities from request data. You can convert a single entity using::
     // Validate and convert to an Entity object
     $entity = $articles->newEntity($this->request->data());
 
+.. note::
```

Comment:
I'm wondering if moving this note here would be enough instead of duplicating it in the same page ..